model/ftn/ww3_bound.ftn

```diff
@@ -308,7 +318,13 @@
               END SELECT !GTYPE
             END IF
           END DO
-
+!
+!/RTD        ! Convert grid boundary cell locations to standard pole
```

Comment:
See comment for equivalent lines in ww3_bounc.ftn

Reply:
See comment above for (ww3_bounc)

Reply:
Addressed in commit https://github.com/ukmo-waves/WW3/pull/10/commits/cc0a20474d3941a1a757690d8973fd763e05fb75